tests/autorestart/test_container_autorestart.py

```diff
@@ -195,27 +195,27 @@ def kill_process_by_pid(duthost, container_name, program_name, program_pid):
                 .format(program_name, container_name))
 
 
-def is_hiting_start_limit(duthost, container_name):
+def is_hiting_start_limit(duthost, service_name, container_name):
```

Comment:
If we did not use the argument `container_name`, then we can remove it?

Reply:
Removed unused argument.